src/Components/DropdownField.res

```diff
@@ -24,13 +34,37 @@ let make = (
     let target = ev->ReactEvent.Form.target
     let value = target["value"]
     setValue(value)
+    if isDisplayValueVisible {
+      let findDisplayValue =
+        options
+        ->Array.find((ele: optionType) => ele.value === value)
+        ->Option.getOr({
+          label: "",
+          value: "",
+          displayValue: "",
+        })
+
+      switch setDisplayValue {
+      | Some(fun) => fun(_ => findDisplayValue.displayValue->Option.getOr(value))
```

Comment:
Please rename this `Some(fun)` to setDisplayValue or some other name

Reply:
Updated